www/js/share/components/accounts/accounts.template.html

```diff
@@ -4,7 +4,7 @@
     </div>
     <select class="account-select" ng-change="vm.updateAccount(vm.selectedAccount)" ng-model="vm.selectedAccount">
             <option ng-repeat="a in vm.accounts" value="{{a.token}}" ng-selected="{{a.token == vm.selectedAccount}}">
-            {{a.id}} {{a.currency == null ? "" : "(" + a.currency + ")"}}
+            {{a.id}} {{!a.currency ? "" : "(" + a.currency + ")"}}
```

Comment:
@nazaninreihani I have checked the app and find out why we have two different result. 
BTW, let add your changes, plz use double exclamation sign instead of one as you know it works better.